button/button.css

```diff
@@ -0,0 +1,123 @@
+body{
+    margin-bottom: 5rem;
+}
+
+.btn-container{
+    display:flex;
+    width:70%;
+    height: 5rem;
+    align-items: center;
+    justify-content:space-evenly;
+}
+
+.btn-item{
+    height: 2.5rem;
+    font-size: 1.2rem;
+    border:none;
+    transition:0.5s;
+    cursor: pointer;
+}
+
+.btn-icon{
+    background-color:var(--white);
+}
+
+.btn-flex{
+    justify-content: center;
+    align-items: center;
+}
+
+.btn-primary-1,
+.btn-primary-3,
+.btn-floating-action{
+    color: var(--white);
+}
+
+.btn-primary-1,
+.btn-primary-3,.btn-link-1,
+.btn-link-2{
+    border-radius:3px;
+}
+
+.btn-home-icon,
+.btn-bag-icon{
+    color: var(--green);
+}
+
+.btn-home-icon:hover,
+.btn-bag-icon:hover{
+    color: var(--dark-green);
+}
+
+.btn-heart-icon,
+.btn-search-icon:hover,
+.btn-primary-3{
+    border-radius: 0.5rem;
+}
```

Comment:
I guess here you should use values in px as it is less than 10px

Reply:
Ok, I have changed that to px